src/onegov/org/utils.py

```diff
@@ -1185,3 +1185,23 @@ def widest_access(*accesses: str) -> str:
         except ValueError:
             pass
     return ORDERED_ACCESS[index]
+
+
+def timestamp_to_seconds(t: str) -> int:
+    """Convert a timestamp to seconds.
+       Examples:
+       '1m30s' -> 90
+       '30s' -> 30
+       '1h2m30s' -> 3750"""
+
+    seconds = 0
+    if 'h' in t:
+        h, t = t.split('h')
+        seconds += int(h) * 3600
```

Comment:
Since this timestamp comes from a user input, we should probably catch errors coming from misspelling. Maybe one try/catch with defaulting to 0?

Reply:
BTW wouldn't this be much easier with regex? You could also add a validator to the form using the same regex

Reply:
Something like ` r'(?:(\d+)h)?(?:(\d+)m)?(?:(\d+)s)?'`